### DISCUSSION TITLE
Бэкапирование базы данных консольной утилитой

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#311] [ОШБК!] [answered]
Здравствуйте при бэкапировании базы данных консольной утилитой выдает ошибку, как ее исправить? 
![image](https://user-images.githubusercontent.com/75322878/212259926-d3a9a997-41d7-40ef-8d29-a70a88fed709.png)

**Comment by erpic-ru** [comment, marked as answer]
Если мы дампируем [multi](https://ru.docs.totum.online/multi-nomulti), то помимо других параметров необходимо указать какую конкретную схему:

bin/totum schema-backup --schema="shema_name" prod_dump.sql

**Comment by Leo-ui-528** [comment]
Спасибо
